_data/changelogs/site-alert.yml

```diff
@@ -34,6 +39,11 @@ items:
     githubPr: 4106
     githubRepo: uswds
     versionUswds: 2.11.1
+  - date: 2021-03-17
+    summary: Updated and improved guidance for site alert.
```

Comment:
```suggestion
    summary: Updated and improved guidance for `site-alert`.
```

Reply:
I would like to leave this as is because it is referring to the component name, not the package name.